schema.sql

```diff
@@ -1,5 +1,8 @@
-/* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals (
+    id serial NOT NULL,

```

Comment:
- [ ] [OPTIONAL] Please make the id column not null and autoincremented so you don't need to insert values on it like:
```
id int NOT NULL IDENTITY(1,1)
```
- [ ] [OPTIONAL] Kindly mark the id column as the primary key to uniquely identify rows in the table.
```
PRIMARY KEY (id)
```